src/main/java/org/alliancegenome/curation_api/dao/base/BaseSQLDAO.java

```diff
@@ -111,6 +111,22 @@ public E find(Long id) {
 			return null;
 		}
 	}
+	
+	public List<String> findFilteredIds(Map<String, Object> params) {
+		List<String> primaryKeys = new ArrayList<>();
+		SearchResponse<E> results = findByParams(params);
+		for (E entity : results.getResults()) {
```

Comment:
The only other concern I have with this is if the result set is large. We may not get an indication of why the server is "hanging". This pulls the whole objects and then pulls out the PK. I think there is a way to only ask for the PK. Basically a `select id from Table` vs `select * from Table` may have to go with this for now but might bite us on large result sets.

Reply:
Agreed - at least we now have a shared method that we can update with a better solution though.